netlify.toml

```diff
@@ -0,0 +1,12 @@
+[build]
+  publish = "dist"
+  command = "yarn build"
+
+  [build.environment]
+    VUE_APP_GA_TAG = "UA-157426670-1"
```

Comment:
I don't think we should keep this particular setting in version control.

Reply:
I'm okay with removing it, but FWIW, we shouldn't have any expectation of privacy regarding this value, as it's totally observable on the client. Also, google will reject requests with this tag that aren't from the whitelist we've configured in the console.

Reply:
So, there's no way someone could accidentally send their own analytics data to our data sink? That's more what I'm concerned about, rather than privacy. If that's the case, we can leave this as-is.

Reply:
If configured right I believe so, however I'm not sure if it currently is, or if I have the permissions to ensure that, so for now I'll remove it from here.

Reply:
Oh I don't think you need to remove it. We just need to make sure that we have GA set up the right way. I naively assumed that by default, this kind of "leak" wouldn't be protected against, but any thought at all shows that Google must have built in this protection, for fear of making their analytics product completely useless to people.